### PR TITLE
[fix](docs)Correcting spelling errors,add isolated file to siderbars.

### LINKED ISSUE #31383: [Doris-Future](docs) Isolated documents detection and fix
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/doris/issues?q=is%3Aissue) and found no similar issues.

### Description

Nowadays our [document](https://doris.apache.org/docs/get-starting/quick-start/) has many isolated pages like https://doris.apache.org/zh-CN/docs/dev/sql-manual/sql-reference/Data-Definition-Statements/Create/CREATE-JOB (isolated means they are not included by sidebar so we can't jump to them from sidebar. but could by searching). We need to link them to proper category.

### Solution

the documents' organization is all in `docs/sidebars.json`. you can write a script easily (maybe with helping of gpt) to travel `docs/` directory and find all `*.md` files. then compare them with `sidebars.json` to figure out which ones are not included properly. then fix the `sidebars.json`. put the missing ones into right category.

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by CSTGluigi**
Correcting spelling errors,add isolated file to siderbars.
Issue Number: close #xxx
partially fix #31383 
This submission only adds existing . md files to siderbars.json, so some pages will not be able to jump properly because some files have not been translated yet.

just add something and rename

### CODE REVIEW

**Comment by CSTGluigi** [comment]
run buildall

**Comment by zclllyybb** [comment]
basically it's right. but there's still something to be aware of:
1. please change your PR's title to fit the Doris' style. you could see other PR's.
2. for the added docs, make sure they have proper header like 
```md
---
{
    "title": "XXX",
    "language": "en" or "zh-CN"
}
---
```
3. make sure the title in md file(like `# xxx` and "title: XXX" in header) is same word with filename. and with the right spelling and the right capital/lower cases.
4. since it's a change not great, you could keep this pr a draft until you have deal all the isolated docs. then set it ready for review and then we consider to merge it.

thanks for your contribution~

**Comment by CSTGluigi** [comment]
> basically it's right. but there's still something to be aware of:
> 
> 1. please change your PR's title to fit the Doris' style. you could see other PR's.
> 2. for the added docs, make sure they have proper header like
> 
> ```
> ---
> {
>     "title": "XXX",
>     "language": "en" or "zh-CN"
> }
> ---
> ```
> 
> 3. make sure the title in md file(like `# xxx` and "title: XXX" in header) is same word with filename. and with the right spelling and the right capital/lower cases.
> 4. since it's a change not great, you could keep this pr a draft until you have deal all the isolated docs. then set it ready for review and then we consider to merge it.
> 
> thanks for your contribution~

i think i have done.
Is this push in a ready state?
cause it shows that “Code Checks / Clang Tidy (pull_request_target) Skipped”

File: docs/sidebarsCommunity.json [hunk 1]
```diff
@@ -68,7 +68,8 @@
                 "developer-guide/how-to-share-blogs",
                 "developer-guide/bitmap-hll-file-format",
                 "developer-guide/github-checks",
-                "developer-guide/regression-testing"
+                "developer-guide/regression-testing",
+                "developer-guide/minidump"
```
Comment (zclllyybb) [inline docs/sidebarsCommunity.json:72]: This item could be removed since it's not useful anymore. also its' md files.

**Review by zclllyybb — COMMENTED** [review]
almost good. only one thing to fix

**Comment by zclllyybb** [comment]
run buildall

**Review by zclllyybb — APPROVED** [review]
LGTM

**Review by BiteTheDDDDt — DISMISSED** [review]

**Comment by zclllyybb** [comment]
run buildall

**Comment by zclllyybb** [comment]
run buildall

**Review by zclllyybb — APPROVED** [review]

**Review by BiteTheDDDDt — APPROVED** [review]

**Pull request merged**